5.0/fr/0x24-V15-Secure-Coding-and-Architecture.md

```diff
@@ -0,0 +1,77 @@
+# V15 Développement et architecture sécurisés
+
+## Objectif de contrôle
+
+De nombreuses exigences pour établir une architecture sécurisée et défendable dépendent d’une documentation claire des décisions prises concernant la mise en œuvre de contrôles de sécurité spécifiques et des composants utilisés dans l’application.
```

Comment:
This sentence does not appear to map to the original text: "Many ASVS requirements either relate to a particular area of security, such as authentication or authorization, or pertain to a particular type of application functionality, such as logging or file handling."